src/corelibs/U2View/src/ov_mca/McaEditorWgt.cpp

```diff
@@ -56,7 +56,7 @@ McaEditorWgt::McaEditorWgt(McaEditor* editor)
     initWidgets();
 
     refArea = new McaEditorReferenceArea(this, getEditor()->getReferenceContext());
-    connect(refArea, SIGNAL(si_selectionChanged()), statusBar, SLOT(sl_update()));
+    connect(refArea, SIGNAL(si_selectionChanged()), statusBar, SLOT(sl_updateStatusBar()));
```

Comment:
Use function references for signal/slot connection for all changed cases. This will ensure we have no bugs during the compile time